src/pages/index.tsx

```diff
@@ -1,21 +1,30 @@
-import GrantsList from '@/components/grantslist';
-import Link from 'next/link';
+import GrantsList from "@/components/grantslist";
+import Link from "next/link";
+import { useState } from "react";
 
 export default function Home() {
+  const [search, setSearch] = useState("");
 
   return (
-    <div className="container mx-auto p-4">
-      <div className="flex items-center justify-between mb-6">
-        <h1 className="text-3xl font-bold">Grants</h1>
-        <Link href="/proposals/create" >
-          <button
-            className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
-          >
-            Create Proposal
-          </button>
-        </Link>
+    <div className="w-screen px-14 py-4">
+      <div className="flex justify-between flex-col gap-10 mb-7">
+        <h1 className="text-5xl font-bold">Grants</h1>
+        <div className="flex justify-between">
+          <input
```

Comment:
Instead of having the input here and passing the search term as props into GrantsList, consider moving this input into the GrantsList component

Reply:
ngl for this one if we move the input to the grantslist component then this component is pretty much just the title and the grantslist component. We might as well leave it as is or bring everything here.

Reply:
Your right, lets move the grantslist back here unless we will be using it on another page